src/managers/ReactionManager.js

```diff
@@ -1,22 +1,30 @@
 'use strict';
 
-const DataStore = require('./DataStore');
 const MessageReaction = require('../structures/MessageReaction');
+const BaseManager = require('./BaseManager');
 
-/**
- * Stores reactions.
- * @extends {DataStore}
- */
-class ReactionStore extends DataStore {
+class ReactionManager extends BaseManager {
   constructor(message, iterable) {
     super(message.client, iterable, MessageReaction);
+
+    /**
+     * The message this manager belongs to.
```

Comment:
```suggestion
     * The message this manager belongs to
```

Another point of inconsistent capitalisation